lib/shoryuken/environment_loader.rb

```diff
@@ -18,12 +18,12 @@ def initialize(options)
     end
 
     def setup_options
+      initialize_rails if load_rails?
       initialize_options
```

Comment:
Since `EnvironmentLoader#intialize_options` will read in the config file first, then the CLI args, it seems like we can safely remove the call to merge in the CLI args from  `Shoryuken::Runner#run`. What do you think?

https://github.com/ruby-shoryuken/shoryuken/blob/b0dc0c505885be5fcd8c1e31a6a544b0ebc9800d/lib/shoryuken/runner.rb#L33-L34

Reply:
That makes sense. Also, removing this line will clear things up as to when/how the options get loaded and avoids further confusion.